docs/api/console.md

```diff
@@ -40,6 +45,22 @@ pagy_metadata(pagy)
    :size=>[1, 4, 4, 1],
 ...
 ```
++++ rails console
+
+!!!warning Warning
+
+Avoid using `rails console` with `Pagy::Console`.
+
+Use `irb` instead. However, if you need `rails console` please ensure you do not freeze the `Pagy::DEFAULT` hash, in your `pagy.rb` config file, otherwise you'll receive a "can't modify frozen Hash" exception:
```

Comment:
Could we formulate the message suggesting to temporarily comment out the `Pagy::DEFAULT.freeze` in the `pagy.rb`? WIthout that it looks like the freeze is not so useful or not compatible with regular usage.

Reply:
Good point my wording is clumsy.